### PR TITLE
Add InitializeSpacetimeTags mutator

### PR DESCRIPTION
**Pull request by nikwit**
## Proposed changes
Adds a mutator that computes the inverse spacetime metric and the trace of the Christoffel symbols of the second kind. These will be needed for the 2nd order evolution of the worldtube.

### CODE REVIEW

File: src/Evolution/Systems/CurvedScalarWave/Worldtube/SingletonActions/InitializeSpacetimeTags.hpp [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  using simple_tags_from_options = tmpl::list<Tags::ExcisionSphere<Dim>>;
+  using const_global_cache_tags = tmpl::list<>;
+  using mutable_global_cache_tags = tmpl::list<>;
+  using argument_tags = simple_tags_from_options;
+  using simple_tags = tmpl::list<
+      gr::Tags::InverseSpacetimeMetric<Dim, Frame::Grid, double>,
+      gr::Tags::TraceSpacetimeChristoffelSecondKind<Dim, Frame::Grid, double>>;
+  using return_tags = simple_tags;
+
+  static auto apply(const gsl::not_null<tnsr::AA<double, Dim, Frame::Grid>*>
```
Comment (nilsdeppe) [inline src/Evolution/Systems/CurvedScalarWave/Worldtube/SingletonActions/InitializeSpacetimeTags.hpp]: move definition to cpp?

File: src/Evolution/Systems/CurvedScalarWave/Worldtube/SingletonActions/InitializeSpacetimeTags.hpp [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+#include "Utilities/Gsl.hpp"
+
+namespace CurvedScalarWave::Worldtube::Initialization {
+
+/*!
+ * \brief Initializes the inverse spacetime metric and trace of the spacetime
+ * christoffel symbol in the co-rotating grid frame at the position of the
+ * scalar charge.
+ *
+ * \details This assimes a circular orbit so the spacetime quantities are
```
Comment (nilsdeppe) [inline src/Evolution/Systems/CurvedScalarWave/Worldtube/SingletonActions/InitializeSpacetimeTags.hpp]: `assimes` -> `assumes`?

**Review by nilsdeppe — CHANGES_REQUESTED** [review]
LGTM, couple small suggestions. Please squash immediately :)

**Comment by nikwit** [comment]
thank you, I squashed it in

**Review by nilsdeppe — APPROVED** [review]

**Comment by nilsdeppe** [comment]
CI failure is unrelated timeout

**Pull request merged**